reservation/views.py

```diff
@@ -137,42 +136,6 @@ def confirm_reservation(request, reservations, schedule):
     )
     msg.send()
 
-    @login_required
```

Comment:
Why did you delete these views?

Reply:
I accidentally copied them twice. In that case, the indentation is wrong